packages/layout/src/css/page-width-layout.js

```diff
@@ -0,0 +1,17 @@
+import * as core from '@pluralsight/ps-design-system-core'
```

Comment:
Probably a good idea to use this syntax here `import { layout } from '@pluralsight/ps-design-system-core'`. We want to stop using star to ensure effective treeshaking